MathProblems/2523_Closest_Prime_Numbers_in_Range.py

```
@@ -0,0 +1,92 @@
+import sys
+from sympy import *
```

Ruff (0.11.9)

`from sympy import *` used; unable to detect undefined names

(F403)